web/src/compose.js

```diff
@@ -52,19 +52,21 @@ export function clear_preview_area() {
     $("#compose .preview_mode_disabled .compose_control_button").attr("tabindex", 0);
 }
 
-export function show_preview_area() {
-    // Disable unneeded compose_control_buttons as we don't
-    // need them in preview mode.
-    $("#compose").addClass("preview_mode");
-    $("#compose .preview_mode_disabled .compose_control_button").attr("tabindex", -1);
+export function show_preview_area(init = true) {
```

Comment:
I think making a `rerender_preview_area` function that does that work, and then having `show_preview_area` call it as a subroutine, would be nicer than this `init=true` function.